### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by verissimohenry**
In this milestone, I followed the provided design specs to build a UI for your app.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,25 +1,58 @@
 /* eslint-disable react/prop-types */
```
Comment (rindrajosia) [inline src/components/Book.js]: - [x] [REQUIRED] Please don't disable react/prop-types  validation. Keep in mind that PropTypes are a mechanism to ensure that components use the correct data type and pass the right data, and that components use the right type of props, and that receiving components receive the right type of props.

File: src/index.css [hunk 2]
```diff
@@ -1,27 +1,258 @@
...
+.title {
+  margin: 6px 47px 2px 0;
+  font-size: 30px;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.navbar-links {
+  display: flex;
+  align-items: center;
```
Comment (rindrajosia) [inline src/index.css]: - [x] [REQUIRED] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: src/index.css [hunk 3]
```diff
@@ -1,27 +1,258 @@
...
+  line-height: normal;
+}
+
+.App {
+  width: 90%;
+}
+
+.navbar {
+  width: 100%;
+  display: flex;
```
Comment (rindrajosia) [inline src/index.css:28]: - [x] [REQUIRED] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: src/index.css [hunk 4]
```diff
@@ -1,27 +1,258 @@
...
+  justify-content: space-around;
+  align-items: center;
```
Comment (rindrajosia) [inline src/index.css:29]: - [x] [REQUIRED] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [x] Files not related to the project should not be committed and added to .gitignore (e.g. .vscode)([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [x] all element in the design, features that haven’t been implemented yet should be added as a placeholder and does not need to be implemented ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone5/designAllElementAsPlaceholder.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/45791599/)).


- [x] I was not able to check how each book will be displayed in your APP because no default list of books is shown. In addition, I was not able to add new book. It will be nice if you fix it and implement all the features and styles related to it.
- [x] Styling should be **EXACTLY** the same as the one provided in the designs ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone5/exactlySameAsDesign.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/45791599/)) (eg: font-color, font-size, font-weight, layout, margin and padding between each elements, ...)


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- It will be nice if you add a live version of the last update of your APP in the readme file.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @verissimohenry, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [x] Kindly add `.vscode` folder into the `.gitignore` file and ensure that it is not uploaded into the repo as suggested in the previous review.
- [x] Your design still doesn't match the given one. Kindly check the screenshot and ensure that the avatar, item container, and add_book button align
ddd
 
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 5]
```diff
@@ -10,6 +10,7 @@ node_modules
 
 # production
 /build
+.vscode
```
Comment (george-shammar) [inline .gitignore:14]: - [ ] Although you've added `.vscode` to `.gitignore`, it is still visible on github. You need to manually remove it. Please, let's remove the file from the repo as mentioned in the previous reviews. :pray:

File: README.md [hunk 6]
```diff
@@ -2,6 +2,7 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+![screenshot](src/bo.png)
 ## Available Scripts
 
 In the project directory, you can run:
```
Comment (george-shammar) [inline README.md:8]: [OPTIONAL]

- Please include the live version of your application in your README.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @verissimohenry , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by george-shammar — APPROVED** [review]
Hi @verissimohenry , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**